src/components/Homepage/Container.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import HomepageList from "./List";
+import Button from "./Button";
+import Description from "./Description";
+import Logo from "./Logo";
+import styled from "@emotion/styled";
+import background from "../../static/svg/group-13.svg";
+
+const BackgroundImage = styled.div`
+  background-image: url(${background});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+`;
+const Container = () => (
```

Comment:
I think this component should have better name

Reply:
fixed